pkg/aggregator/apiserver/plugin/admission/mutation.go

```diff
@@ -0,0 +1,43 @@
+package admission
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/mattbaird/jsonpatch"
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func FromMutationResponse(current []byte, r *backend.MutationResponse) (*admissionv1.AdmissionReview, error) {
+	res := admissionv1.AdmissionResponse{
+		Allowed: r.Allowed,
+		Result: &metav1.Status{
```

Comment:
IIUC, the result is only needed when `!r.Allowed` and in that case we also do not need to do anything with the patch

Reply:
👍 i'll double check this and remove it if it's not needed

Reply:
i added a check for this